examples/lib/defaultHtmlStepUi.js

```diff
@@ -182,14 +184,14 @@ function DefaultHtmlStepUi(_sequencer, options) {
       }
     }
     else {
-      $('#load-image').append(step.ui);
+      $('#load-image').append(step.ui); // Default UI without extra tools for the first step(load image).
```

Comment:
.

Reply:
It is only subject/predicate then no need of .

Reply:
```suggestion
      $('#load-image').append(step.ui); // Default UI without extra tools for the first step(load image)
```